superset/app.py

```diff
@@ -552,7 +552,6 @@ def configure_fab(self) -> None:
         appbuilder.indexview = SupersetIndexView
         appbuilder.base_template = "superset/base.html"
         appbuilder.security_manager_class = custom_sm
-        appbuilder.update_perms = False
```

Comment:
This is already `False` per [here](https://github.com/apache/incubator-superset/blob/3a08fd04f3f9ebaaa48fa554028c26062de42f02/superset/extensions.py#L135) which is set by FAB either [here](https://github.com/dpgaspar/Flask-AppBuilder/blob/120bc3ca38c4559a0099fe3b1a1badb319b6546e/flask_appbuilder/base.py#L145) and potentially reset [here](https://github.com/dpgaspar/Flask-AppBuilder/blob/120bc3ca38c4559a0099fe3b1a1badb319b6546e/flask_appbuilder/base.py#L186-L187), though isn't since `self.update_perms` is already `False`.